frontend/src/scenes/exports/ExportsList.tsx

```diff
@@ -103,6 +104,16 @@ export function Exports(): JSX.Element {
                                 error: resumeError,
                             } = useExportAction(currentTeamId, export_.id, 'unpause')
 
+                            const deleteCallback = useCallback(() => {
```

Comment:
I _think_ we'd want this callback to very with index 🤔  Perhaps best to not even use `useCallback` here?

Reply:
Not using `useCallback` anymore!